yaml/com/ibm/PLDM/ChapData.interface.yaml

```diff
@@ -0,0 +1,15 @@
+description: >
+    Implement this interface to pass Chap encrypted string to Host from PLDM.
```

Comment:
Please change like this "Implement this interface to pass Chap data details to Host which are set by the user."